### PR TITLE
Add AlmaLinux support

### PR DESCRIPTION
**Pull request by EMH333**

### CODE REVIEW

**Review by ramereth — CHANGES_REQUESTED** [review]
- Update logic in `osl-repos-test::disable` and `osl-repos-test::highavailability' so that it'll work on alma as well.

File: kitchen.yml [hunk 1]
```diff
@@ -8,10 +8,23 @@ provisioner:
...
+driver:
+  flavor_ref: 'm1.medium'
+
 suites:
+  - name: alma
+    run_list:
+      - recipe[osl-repos::alma]
+    excludes:
+      - centos-7
+      - centos-stream-8
```
Comment (ramereth) [inline kitchen.yml:20]:
```suggestion
      - centos_stream-8
```

File: resources/alma.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  end
+
+  passthrough = {
+    'exclude' => (new_resource.exclude.join(' ') unless new_resource.exclude.empty?),
+  }.compact
+
+  yum_alma_baseos 'default' do
+    mirrorlist nil
+    baseurl "#{alma_url}/#{release_var}/BaseOS/#{base_arch}/os/"
+    passthrough passthrough
```
Comment (ramereth) [inline resources/alma.rb:31]: This is generally bad practice and could make some debugging an issue later on. Please rename the variable to something else.

**Review by ramereth — CHANGES_REQUESTED** [review]

File: kitchen.yml [hunk 3]
```diff
@@ -8,10 +8,23 @@ provisioner:
...
+  flavor_ref: 'm1.medium'
+
 suites:
+  - name: alma
+    run_list:
+      - recipe[osl-repos::alma]
+    excludes:
+      - centos-7
+      - centos_stream-8
+
```
Comment (ramereth) [inline kitchen.yml:21]: Could you please remove the new line here and on line 27?

**Review by ramereth — APPROVED** [review]
Major bump please.

**Comment by EMH333** [comment]
!bump major ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a major-level version bump to v3.0.0.  Have a nice day!

**Pull request merged**